### PR TITLE
Django Tutorial Part 6: Generic list and detail views

### PR DESCRIPTION
**Pull request by khanhlyvu30**
ready

### CODE REVIEW

**Comment by khanhlyvu30** [comment]
ready

**Comment by khanhlyvu30** [comment]
ready

File: locallibrary/catalog/templates/base_generic.html [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <!DOCTYPE html>
 <html lang="en">
+<<<<<<< HEAD
```
Comment (thaodtd-0665) [inline locallibrary/catalog/templates/base_generic.html]: còn conflict code nè em, fix hết nhé

File: locallibrary/catalog/templates/catalog/author_detail.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      <p class="{% if book.status == 'a' %}text-success{% elif copy.status == 'm' %}text-danger{% else %}text-warning{% endif %}">
+        {{ book.get_status_display }}
+      </p>
+      {% if book.status != 'a' %}
+        <p> <a href="{{ book.get_absolute_url }}">{{book.title}}</a></p>
+      {% endif %}
+      <p> {{ book.summary }}</p>
+    {% endfor %}
+  </div>
+{% endblock %}
```
Comment (thaodtd-0665) [inline locallibrary/catalog/templates/catalog/author_detail.html]: lỗi end line

File: locallibrary/catalog/templates/catalog/author_list.html [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+        (<a href="{{ author.get_absolute_url }}">{{ author.date_of_birth}}</a> -
+        <a href="{{ author.get_absolute_url }}">{{ author.date_of_death}}</a>)
+
+      </li>
+    {% endfor %}
+  </ul>
+  {% else %}
+    <p>There are no author in the library.</p>
+  {% endif %}       
+{% endblock %}
```
Comment (thaodtd-0665) [inline locallibrary/catalog/templates/catalog/author_list.html]: end line

File: locallibrary/catalog/views.py [hunk 4]
```diff
@@ -24,4 +26,40 @@ def index(request):
     }
 
     # Render the HTML template index.html with the data in the context variable
-    return render(request, 'index.html', context=context)
\ No newline at end of file
+<<<<<<< HEAD
```
Comment (thaodtd-0665) [inline locallibrary/catalog/views.py]: conflict code

**Comment by khanhlyvu30** [comment]
ready

**Review by thaodtd-0665 — APPROVED** [review]

**Comment by thaodtd-0665** [comment]
merged

**Pull request merged**